templates/templates/navigation-download-h.html

```diff
@@ -14,6 +14,9 @@ <h4 class="p-heading-four is-dense"><a href='/download/server'>Ubuntu Server&nbs
           <a href="/download/server/thank-you?version={{ releases.lts.full_version }}&amp;architecture=amd64" class="p-button--positive p-button--small" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Server', 'eventLabel' : '{{ releases.lts.short_version }} LTS', 'eventValue' : undefined });">{{ releases.lts.short_version }} LTS</a>
           {% if releases.latest.short_version > releases.lts.short_version %}<a href="/download/server/thank-you?version={{ releases.latest.full_version }}&amp;architecture=amd64" class="p-button--neutral p-button--small" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Server', 'eventLabel' : '{{ releases.latest.short_version }}', 'eventValue' : undefined });">{{ releases.latest.short_version }}</a>{% endif %}
           <ul class='p-text-list--small is-bordered' >
+            <li class='p-list__item'><a class='p-link' href='https://multipass.run'>
+              Mac and Windows
```

Comment:
do you think 'Ubuntu VMs on Mac and Windows' might be better?

Reply:
It may get more click-through, but I thought it was more consistent with the other items: "ARM", "IBM Power", etc. (implied "Run Ubuntu on...").

![Screenshot from 2020-05-26 17-20-47](https://user-images.githubusercontent.com/18480003/82918628-4a043080-9f75-11ea-84be-96d21646eda9.png)
![Screenshot from 2020-05-26 17-20-20](https://user-images.githubusercontent.com/18480003/82918638-4bcdf400-9f75-11ea-9614-fe8fd69cc3d3.png)

Thoughts?

Reply:
Feels like an A/B test, but for now, let's go with your preference.